demo/Makefile

```diff
@@ -27,7 +27,7 @@ local_retention_ms=1000 # 1 second
 
 .PHONY: create_topic
 create_topic:
-	docker exec $(kafka_container) \
+	docker exec -e KAFKA_OPTS= $(kafka_container) \
```

Comment:
I suspect `KAFKA_OPTS` is taken into account only when a real Kafka is started via the `/etc/confluent/docker/configure` script, so this probably can be skipped here and below

Reply:
Nope, it's picked by the `kafka-run.sh` command, some it tries to initialize the agent on every execution (unfortunately).

Reply:
Yeah, you're right. It's Kafka's own thing, not Conlfuent's.

In this case, could you please add the same to `create_topic_ts_by_time`, `create_topic_no_ts_by_size`, `create_topic_no_ts_by_time`?

Reply:
Of course 😅 good catch